code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -183,13 +188,23 @@
 		M.Dizzy(5)
 		if(iscultist(M) && prob(5))
 			M.say(pick("Av'te Nar'sie","Pa'lid Mors","INO INO ORA ANA","SAT ANA!","Daim'niodeis Arc'iai Le'eones","R'ge Na'sie","Diabo us Vo'iscum","Eld' Mon Nobis"))
+		else if(is_servant_of_ratvar(M) && prob(5))
+			switch(pick("speech", "message", "emote"))
+				if("speech")
+					M.say("...[pick("Ratvar... lbhe yvtug tebjf qnex", "Jurer ner lbh, znfgre?", "Ur yvrf ehfgvat va Reebe", "Chetr nyy hagehguf naq... naq... fbzrguvat")]...")
+				if("message")
+					M << "<span class='warning'><b>[pick("Ratvar's illumination of your mind has begun to flicker.", "He lies rusting in Reebe, derelict and forgotten. And there he shall stay.", \
+					"You can't save him. Nothing can save him now.", "It seems that Nar-Sie will triumph after all.")]</b></span>"
```

Comment:
From the universal viewpoint, clockcult is in no way a "good" force in the lore. One could say that both cults are corrupted, but in different ways. The clockcult think they are good guys, and the knowledge granted them by Ratvar proves it. This is one of the differences between two cults - Ratvar takes some notice of his cultists, "converting" them to his point of view, while Nar'sie treats them only as mechanisms enabling him to arrive. You can see the difference in conversion - Ratvarians pour the knowledge (as biased as it may be) into their followers, and treat it as the Light of Ratvar, while Nar'sians just bend their wills towards the goals of their god, and fill them with an infinitely small ammount of his power.
It's hard to say which one is more horrifying - the cult of Nar'sie messes with your head, so that you do as Nar'sie wants, while the clockcult doesn't do that - it instead fills you with knowledge that instantly turns you into a zealot. I believe that Ratvarians still have free will, and that the information they are given is instead made undeniable, and resulting viewpoint is that serving Ratvar is the only way.
Deconversion is similar in both cases - for Nar'sians, it removes the corruption the blood magic has planted into their minds, restoring their minds at the cost of several memories, that if left, would prevent reconversion. To Ratvarians, it instead removes the enlightenment, the thing clockcult could see as the greatest gift of their god - knowledge. It could slowly dissolve the deep conviction that clockcult has. That's why they start doubting Ratvar - as the undeniable truth planted into their minds disappears, but the memories of their time as clockcult remain, they start asking questions. The displays of Justicar' power take on new meaning - how desperate must Ratvar be, if his generals interact with such mere mortals. All clockcult structures are MADE from rusted fragments of Ratvar - a rusted god! If we interpret "Light" as "Knowledge", the lines about the light fading take on this new sense.
From crewmember point of view, Ratvar and Nar'sie are hardly different. Both cults use magic, both convert others - either by bending their will outright, or by implanting something as unnatural as undeniable truth, essentially instantly turning you into a zealot. Both want to bring destructive forces into the world. And though their intentions differ, this makes no real difference to the small men. 

In other words, Clockcult is not "Good" - they are zealots who THINK that they are good. Nar'sians are just too corrupted to care.